code/modules/mob/living/silicon/robot/robot_defense.dm

```diff
@@ -341,36 +341,39 @@ GLOBAL_LIST_INIT(blacklisted_borg_hats, typecacheof(list( //Hats that don't real
 		balloon_alert(user, "expose the fires first!")
 		return FALSE
 
-	balloon_alert(user, "interface hacked")
-	emag_cooldown = world.time + 100
-
-	if(connected_ai && connected_ai.mind && connected_ai.mind.has_antag_datum(/datum/antagonist/malf_ai))
-		to_chat(src, span_danger("ALERT: Foreign software execution prevented."))
-		logevent("ALERT: Foreign software execution prevented.")
-		to_chat(connected_ai, span_danger("ALERT: Cyborg unit \[[src]\] successfully defended against subversion."))
-		log_silicon("EMAG: [key_name(user)] attempted to emag cyborg [key_name(src)], but they were slaved to traitor AI [connected_ai].")
-		return TRUE // emag succeeded, it was just counteracted
-
-	if(shell) //AI shells cannot be emagged, so we try to make it look like a standard reset. Smart players may see through this, however.
-		to_chat(user, span_danger("[src] is remotely controlled! Your emag attempt has triggered a system reset instead!"))
-		log_silicon("EMAG: [key_name(user)] attempted to emag an AI shell belonging to [key_name(src) ? key_name(src) : connected_ai]. The shell has been reset as a result.")
-		ResetModel()
-		return TRUE
+	balloon_alert(user, "hacking interface...") //BUBBER EDIT: CHANGES THIS DESCRIPTION
+	//BUBBER EDIT BEGIN: NERF NO-TALK EMAGS
+	if(do_after(user, 2 SECONDS))
+		emag_cooldown = world.time + 100
+
+		if(connected_ai && connected_ai.mind && connected_ai.mind.has_antag_datum(/datum/antagonist/malf_ai))
+			to_chat(src, span_danger("ALERT: Foreign software execution prevented."))
+			logevent("ALERT: Foreign software execution prevented.")
+			to_chat(connected_ai, span_danger("ALERT: Cyborg unit \[[src]\] successfully defended against subversion."))
+			log_silicon("EMAG: [key_name(user)] attempted to emag cyborg [key_name(src)], but they were slaved to traitor AI [connected_ai].")
+			return TRUE // emag succeeded, it was just counteracted
+
+		if(shell) //AI shells cannot be emagged, so we try to make it look like a standard reset. Smart players may see through this, however.
+			to_chat(user, span_danger("[src] is remotely controlled! Your emag attempt has triggered a system reset instead!"))
+			log_silicon("EMAG: [key_name(user)] attempted to emag an AI shell belonging to [key_name(src) ? key_name(src) : connected_ai]. The shell has been reset as a result.")
+			ResetModel()
+			return TRUE
```

Comment:
You will make my future time with conflicts much easier by using a 
```
if(!do_after(user, 2 SECONDS))
	return
```
and preventing the extra indent

Reply:
Where do you want this, exactly? I'll do it soon